### PR TITLE
Typos in includes/calendar-era.md

### PR DESCRIPTION
**Pull request by alexb5dh**
basaed -> based
calendard -> calendard

### CODE REVIEW

File: includes/calendar-era.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
+>  Eras in the Japanese calendars are based on the emperor's reign and are therefore expected to change. For example, May 1, 2019 marked the beginning of the Reiwa era in the <xref:System.Globalization.JapaneseCalendar> and <xref:System.Globalization.JapaneseLunisolarCalendar>. Such a change of era affects all applications that use these calendars. See [Handling a new era in the Japanese calendar in .NET](https://devblogs.microsoft.com/dotnet/handling-a-new-era-in-the-japanese-calendar-in-net/) for more information and to determine whether your applications are affected. See [Prepare your application for the Japanese era change](/windows/uwp/design/globalizing/japanese-era-change) for information on testing your applications on Windows systems to ensure their readiness for the era change. See [Working with eras](~/docs/standard/datetime/working-with-calendars.md#working-with-eras) for features in .NET that support calendar with multiple eras and for best practices when working with calendars that support multiple eras.
```
Comment (rpetrusha) [inline includes/calendar-era.md]:
```suggestion
>  Eras in the Japanese calendars are based on the emperor's reign and are therefore expected to change. For example, May 1, 2019 marked the beginning of the Reiwa era in the <xref:System.Globalization.JapaneseCalendar> and <xref:System.Globalization.JapaneseLunisolarCalendar>. Such a change of era affects all applications that use these calendars. See [Handling a new era in the Japanese calendar in .NET](https://devblogs.microsoft.com/dotnet/handling-a-new-era-in-the-japanese-calendar-in-net/) for more information and to determine whether your applications are affected. See [Prepare your application for the Japanese era change](/windows/uwp/design/globalizing/japanese-era-change) for information on testing your applications on Windows systems to ensure their readiness for the era change. See [Working with eras](~/docs/standard/datetime/working-with-calendars.md#working-with-eras) for features in .NET that support calendars with multiple eras and for best practices when working with calendars that support multiple eras.
```

**Comment by rpetrusha** [comment]
Thanks, @alexb5dh, for correcting these two typos. I've suggested one change -- "calendar" should be plural rather than singular.

**Comment by alexb5dh** [comment]
Thanks, fixed.

**Comment by rpetrusha** [comment]
Thanks for the additional change, @alexb5dh. We'll merge your PR as soon as our internal build completes successfully.

**Pull request merged**